src/PlaidLink.js

```diff
@@ -147,16 +134,35 @@ class PlaidLink extends Component {
     }
   }
 
+  // Returns all props but the ones specific to the component plus `children`
+  get htmlProps() {
```

Comment:
Added this because I wanted a behavior similar to what Babel does with object destructuring but wasn't sure if adding a new dependency was okay. It's working fine as is.